doc-content/kogito-docs/src/main/asciidoc/creating-running/proc_kogito-designing-app-rule-units.adoc

```diff
@@ -0,0 +1,89 @@
+[id='proc_kogito-designing-app-rule-units']
+
+= Using DRL rule units as an alternative decision service
+
+As an alternative to using Decision Model and Notation (DMN) to define this example decision service, you can also use a Drools Rule Language (DRL) file implemented as a rule unit.
+
+Rule units are groups of data sources, global variables, and DRL rules that function together for a specific purpose. You can use rule units to partition a rule set into smaller units, bind different data sources to those units, and then execute the individual unit. Rule units are an enhanced alternative to rule-grouping DRL attributes such as rule flow groups, rule agenda groups, or activation groups for execution control. For more information about rule units, see
+ifdef::KOGITO[]
+https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.6/html-single/designing_a_decision_service_using_drl_rules/index#drl-rule-units-con_drl-rules[_{DRL_RULES}_].
+endif::[]
+ifdef::KOGITO-COMM[]
+the https://docs.jboss.org/drools/release/latestFinal/drools-docs/html_single/#drl-rule-units-con_decision-engine[Drools documentation].
+endif::[]
+
+.Procedure
+. In the `src/main/resources/org/acme/kogito` folder that you created for this example, instead of using a DMN file, add the following `PersonRules.drl` file:
++
+--
+.Example person DRL file
+[source]
+----
+package org.acme.kogito.model
+unit EvaluatePerson;
```

Comment:
let's reuse PersonRules to be consistent with the file name; or you can use EvaluatePerson.drl

Reply:
Done with all comments, per our gchat. Resolving all.